### PR TITLE
WIP: Add `genimage.sh`, `Config.in` for Grub2, w/ check for Grub.cfg location.

### PR DESCRIPTION
**Pull request by klaases**
WIP only.

### CODE REVIEW

**Comment by klaases** [comment]
Current error that I am encountering:

```
Executing post-image script /mnt/deploy/iso/minikube-iso/board/iso/x86_64/post-image-efi.sh
/mnt/out/buildroot/output/images/genimage-efi.cfg:34: unexpected token '='
```

- Confirmed that genimage-efi.cfg file is correct, according to default configuration on Buildroot repo.

- Noticed that there is no line 34 in the file, must be a cached version.
 
- Going to clear ./out and try again.

File: deploy/iso/minikube-iso/Dockerfile [hunk 1]
```diff
@@ -37,11 +37,22 @@ RUN apt-get update \
...
 		mkisofs \
+		qemu-system-arm \
+		qemu-system-x86 \
+		libconfuse-dev \
 	&& rm -rf /var/lib/apt/lists/*
 
 RUN localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
 ENV LANG en_US.utf8
 
+# Build genimage (required by some targets to generate disk images)
```
Comment (afbjorklund) [inline deploy/iso/minikube-iso/Dockerfile:48]: This is supposed to be built by buildroot, and should **not** be needed on the host.

`BR2_PACKAGE_HOST_GENIMAGE`

Comment (klaases) [reply deploy/iso/minikube-iso/Dockerfile:48]: Sounds good.

**Comment by klaases** [comment]
Closing as this will be included in a separate PR.

**Pull request closed without merging**